policyengine_us/variables/gov/hhs/medicaid/eligibility/categories/is_optional_senior_or_disabled_for_medicaid.py

```diff
@@ -5,38 +5,19 @@ class is_optional_senior_or_disabled_for_medicaid(Variable):
     value_type = bool
     entity = Person
     label = "Seniors or disabled people not meeting SSI rules"
-    documentation = "Whether this person can claim Medicaid through the State's optional pathway for seniors or people with disabilities."
+    documentation = (
+        "Whether this person qualifies for Medicaid through the State's optional "
+        "aged, blind, or disabled pathway (not otherwise SSI-eligible)."
+    )
     definition_period = YEAR
     reference = "https://www.law.cornell.edu/uscode/text/42/1396a#m"
 
     def formula(person, period, parameters):
-        personal_income = person("ssi_countable_income", period)
-        personal_assets = person("ssi_countable_resources", period)
-        tax_unit = person.tax_unit
-        income = tax_unit.sum(personal_income)
-        assets = tax_unit.sum(personal_assets)
-        ma = parameters(
-            period
-        ).gov.hhs.medicaid.eligibility.categories.senior_or_disabled
         is_senior_or_disabled = person("is_ssi_aged_blind_disabled", period)
```

Comment:
Could you make a new variable for the abd criteria for this program? Since this program specifies age 65 separately we should make that its own policy parameter